amethyst_renderer/src/bundle.rs

```diff
@@ -59,7 +61,12 @@ impl<'a, 'b, 'c, B: PipelineBuild<Pipeline = P>, P: 'b + PolyPipeline> SystemBun
             );
         };
         builder.add_thread_local(
-            RenderSystem::build(self.pipe, self.config).chain_err(|| "Renderer error!")?
+            RenderSystem::build(self.pipe, self.config).chain_err(|| "Renderer error!")?,
+        );
+        builder.add(
+            Processor::<SpriteSheet>::new(),
+            "sprite_sheet_processor",
+            &[],
```

Comment:
Is it wise to always add a SpriteSheet processor to the rendering bundle?
When you're making a 3D, you probably don't need this at all.

Reply:
True that, it should be in the consumer application &mdash; I used `loader.load_from_data(..)`.

* [ ] Todo: Move `Processor::<SpriteSheet>` out

Reply:
Just realized, this should still be in the render bundle, but not enabled until the user requests it. Rationale is we shouldn't require everyone who is using the `DrawSprites` pass to create a bundle and paste this in it:

```rust
builder.add(
    Processor::<SpriteSheet>::new(),
    "sprite_sheet_processor",
    &[],
);
```

Ideally the render bundle would be able to inspect the `Pipeline` for any pass that needs `Sprite`s support and enable this automatically.

Reply:
Then we should get that pipeline parser going beforehand. It shouldn't be that hard and would be very helpful.

Reply:
I can't seem to find info relating to pipeline parser in github issues / source code, do you have a link / can you give me a quick rundown on how it's meant to work?

Reply:
I was talking about the idea you just mentioned of inspecting the pipeline from the render bundle. It shouldn't be that hard to implement I think.

Reply:
ohh, so *this is it* :man_dancing:. haha okay I'll have a go